app/notifiers/course/forum/topic_notifier.rb

```diff
@@ -7,7 +7,8 @@ def topic_created(user, course_user, topic)
     return unless email_enabled.regular || email_enabled.phantom
 
     activity = create_activity(actor: user, object: topic, event: :created)
-    activity.notify(course, :feed) unless course_user.phantom?
+    activity.notify(course, :feed) if course_user && !course_user.phantom? &&
+                                      !topic.posts.order(:created_at).first.is_anonymous
```

Comment:
Did you see this comment?
https://github.com/Coursemology/coursemology2/pull/7584/files/a91859bc9058112721a5d718761fcf874ac554a9#r1791161287